File: Day-1/Data Types.cpp
#include <iostream>
#include <iomanip>
#include <limits>

using namespace std;

int main() {
    int i = 4;
    double d = 4.0;
    string s = "HackerRank ";

    // Declare second integer, double, and String variables.
    int n;
    double dec;
    string str;
    
    // Read and save an integer, double, and String to your variables.
    cin >> n >> dec;
    getline(cin>>ws, str);
    
    // Note: If you have trouble reading the entire string, please go back and review the Tutorial closely.
    
    // Print the sum of both integer variables on a new line.
    cout << i+n << endl;
    
    // Print the sum of the double variables on a new line.
    cout << fixed << setprecision(1) << d+dec << endl;
    // Concatenate and print the String variables on a new line
    // The 's' variable above should be printed first.
     
    cout << s+str << endl;

    return 0;

File: Day-1/Data Types.js
process.stdin.resume();
process.stdin.setEncoding('ascii');

var input_stdin = "";
var input_stdin_array = "";
var input_currentline = 0;

process.stdin.on('data', function (data) {
    input_stdin += data;
});

process.stdin.on('end', function () {
    input_stdin_array = input_stdin.split("\n");
    main();    
});

// Reads complete line from STDIN
function readLine() {
    return input_stdin_array[input_currentline++];
}

function main() {
    var i = 4
    var d = 4.0
    var s = "HackerRank "
    // Declare second integer, double, and String variables.
    var n;
    var dec;
    var str;
    
    // Read and save an integer, double, and String to your variables.
    n = readLine();
    dec = readLine();
    str = readLine();

    // Print the sum of both integer variables on a new line.
    console.log(Number(i)+Number(n));

    // Print the sum of the double variables on a new line.
    console.log((Number(d)+Number(dec)).toFixed(1));

    // Concatenate and print the String variables on a new line
    // The 's' variable above should be printed first.
    console.log(s+str);

}